### DISCUSSION TITLE
Tight integration

### DISCUSSION THREAD

**Discussion by juliajing72** [rodralez/NaveGo#55] [Ideas]
I  really like this code, which is easy to use and debug. Just curious whether the authors plan to write a tight integration version? I realise there are not many tight integration open source codes out there. 
Thank you!

**Comment by rodralez** [comment]
Hello,

Tightly-coupled integration is the first feature on the roadmap of NaveGo to be developed. I hope it will be implemented during this year. 

But, before this we have to implement another function.

Since tightly-coupled integration uses pseudo-ranges and pseudo-range rates, we need a RINEX parser to extract these measurements from a RINEX file.

Any help on getting a RINEX parser or developing one will be appreciated.

Best regards.

**Reply by juliajing72** [reply]
Hello, Thanks for your response. I believe there are quite a few RINEX parsers in Matlab that is available to use. Actually, I have been looking into some recently. For example, this report contains codes on reading the header and data for RINEX 2.11. There's a few bugs which I ran into, but were not difficult to overcome. 
https://www.researchgate.net/publication/303314368_Reading_RINEX_211_Observation_Data_Files
